app/javascript/packages/document-capture/components/acuant-capture.jsx

```diff
@@ -11,27 +11,75 @@ import DataURLFile from '../models/data-url-file';
 /**
  * @typedef AcuantCaptureProps
  *
- * @prop {string}                        label      Label associated with file input.
- * @prop {string=}                       bannerText Optional banner text to show in file input.
- * @prop {DataURLFile=}                  value      Current value.
- * @prop {(nextValue:DataURLFile)=>void} onChange   Callback receiving next value on change.
- * @prop {string=}                       className  Optional additional class names.
+ * @prop {string}                        label                 Label associated with file input.
+ * @prop {string=}                       bannerText            Optional banner text to show in file
+ *                                                             input.
+ * @prop {DataURLFile=}                  value                 Current value.
+ * @prop {(nextValue:DataURLFile)=>void} onChange              Callback receiving next value on
+ *                                                             change.
+ * @prop {string=}                       className             Optional additional class names.
+ * @prop {number=}                       minimumGlareScore     Minimum glare score to be considered
+ *                                                             acceptable.
+ * @prop {number=}                       minimumSharpnessScore Minimum sharpness score to be
+ *                                                             considered acceptable.
+ * @prop {number=}                       minimumFileSize       Minimum file size (in bytes) to be
+ *                                                             considered acceptable.
  */
 
 /**
- * The minimal glare score value to be considered acceptable.
+ * Returns the file size (bytes) of a file represented as a data URL.
+ *
+ * @param {string} dataURL Data URL.
+ *
+ * @return {number} File size, in bytes.
+ */
+export function getDataURLFileSize(dataURL) {
+  const [header, data] = dataURL.split(',');
+  const isBase64 = /;base64$/.test(header);
+  const decodedData = isBase64 ? window.atob(data) : decodeURIComponent(data);
```

Comment:
I was about to write some ruby code to parse these URLs, this was a good reminder the `base64` part is optional, thank you!